review/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.urls import reverse
+# from PIL import Image
+
+
+class Rating(models.TextChoices):
+    ONE_STAR = ('1', 'One Star')
+    TWO_STARS = ('2', 'Two Stars')
+    THREE_STARS = ('3', 'Three Stars')
+    FOUR_STARS = ('4', 'Four Stars')
+    FIVE_STARS = ('5', 'Five Stars')
+    UNSPECIFIED = ('UN', 'Unspecified')
+
+
+class Review(models.Model):
+    # professional_ID = models.ForeignKey(Professional, on_delete=models.SET_NULL)
```

Comment:
Please remove the lines
Or just use these variables, as string until main will contain the relevant ORM for Client/Pro